pkg/asset/internal/templates.go

```diff
@@ -1034,11 +1039,23 @@ metadata:
 data:
   cni-conf.json: |
```

Comment:
@diegs 
Looks like the cni config file name should have `.conflist` suffix to load list of plugins
https://github.com/kubernetes/kubernetes/blob/master/pkg/kubelet/network/cni/cni.go#L110-L127